### PR TITLE
Added clip and ship notebook

### PR DESCRIPTION
**Pull request by nateweisiger**
Also normalized internal directories such that they can easily be shared by new notebook.

### CODE REVIEW

File: jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb [hunk 1]
```diff
@@ -0,0 +1,194 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Item Clipping and Download\n",
+    "\n",
+    "This example clips, downloads and unzips a given scene to the AOI provided, returning only the image data contained in with the AOI. To so this, we use the \"Clips\" API.\n",
```
Comment (danabauer) [inline jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb]: link mention of Clips API to https://www.planet.com/docs/reference/clips-api/

File: jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from os import environ\n",
+    "\n",
+    "# Set API key (this should to be an environment variable)\n",
+    "api_key = environ['PL_API_KEY']\n",
```
Comment (danabauer) [inline jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb]: We should be consistent and use PL_API_KEY everywhere. I think we might have PLANET_API_KEY is some examples.

File: jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from tqdm import tqdm\n",
```
Comment (danabauer) [inline jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb]: is tqdm installed in the docker env?

File: jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb (hunk 2 above)
Comment (nateweisiger) [inline jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb]: I think I changed them all to PL_API_KEY yesterday (as well and change the Docker ENV var. If you see the one above anywhere else, let me know.

File: jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb (hunk 3 above)
Comment (nateweisiger) [inline jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb]: yes.

File: jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb (hunk 2 above)
Comment (nateweisiger) [inline jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb]: I just found this: PLANET_API_KEY = os.getenv('PL_API_KEY'). But that is like that by design I think.

File: jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb (hunk 1 above)
Comment (nateweisiger) [inline jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb]: Done

**Pull request merged**